lib/attachProof.js

```diff
@@ -39,12 +39,14 @@ const {Ed25519Signature2020} = require('@digitalbazaar/ed25519-signature-2020');
  * @param {object} [options.authDoc] - Auth DID Doc, required if using
  *   an accelerator service.
  * @param {string} options.mode - Ledger mode ('test', 'live' etc).
+ * @param {string} options.invocationTarget - The invocationTarget
+ *   (should be the ledger id or the id of a record on the ledger).
```

Comment:
I think this should be removed. The `invocationTarget` here should *always* be the DID identified by the record ID/recordPatch target. The caller should not have to specify it.

The accelerator will use a different invocation target, but they will be responsible for that -- and theirs will always be the same as well -- the ID for the veres one ledger.

```suggestion
```